user.go

```diff
@@ -135,3 +135,11 @@ func (s *Session) GetUserAliases() ([]UserAlias, error) {
 	}
 	return aliases, nil
 }
+
+// CreateNewUser creates a new myradio user with the given parameters
+// This consumes one API request.
+func (s *Session) CreateNewUser(formParams map[string][]string) (user *User, err error) {
+	rs := s.post("/user/createoractivate", formParams)
```

Comment:
this should probably be a CreateOrActivateUser method, if it's calling this endpoint - for consistency